docker-compose.yml

```diff
@@ -30,11 +30,6 @@ services:
     environment:
       - AWS_SECRET_ACCESS_KEY
       - AWS_ACCESS_KEY_ID
-      # https://docs.databricks.com/api/latest/authentication.html#generate-a-token
```

Comment:
this way may be desirable since it doesn't affect prod

Reply:
Unfortunately airflow reads from the environment over the settings set in the UI (which is an unexpected imo, applications should start reading values closest to the user's control). The dummy GCP connections can't be overwritten without creating the connection string.